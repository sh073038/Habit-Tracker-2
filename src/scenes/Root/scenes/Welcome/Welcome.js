// @flow

import React from 'react'

import { withStyles } from 'material-ui/styles'
import Card, { CardContent } from 'material-ui/Card'
import Typography from 'material-ui/Typography'

type Props = {
  classes: Object,
}

const styles = {
  card: {
    width: '100%',
  },
  media: {
    alignContent: 'center',
    height: '500px',
    padding: '8px',
  },
}

const Welcome = ({ classes }: Props) => (
  <Card className={classes.card}>
    <CardContent>
      <Typography type="headline" component="h2">
        Oh hi there
      </Typography>
      <Typography paragraph>
        <br />
        All you can do here is set your personal challenges and track em.<br />Start with login.
      </Typography>
    </CardContent>
  </Card>
)

export default withStyles(styles)(Welcome)
