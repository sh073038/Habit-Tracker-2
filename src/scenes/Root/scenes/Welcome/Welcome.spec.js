import React from 'react'
import { shallow } from 'enzyme'

import Welcome from './Welcome'

describe('Welcome', () => {
  it('renders component', () => {
    const wrapper = shallow(<Welcome />)
    expect(wrapper.getElement()).toMatchSnapshot()
    expect(wrapper.props()).toMatchSnapshot()
    expect(wrapper.props()).toEqual({
      classes: { card: 'Welcome-card-1', media: 'Welcome-media-2' },
    })
  })
})
