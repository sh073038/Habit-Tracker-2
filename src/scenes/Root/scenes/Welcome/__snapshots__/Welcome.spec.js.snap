// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders component 1`] = `
<Welcome
  classes={
    Object {
      "card": "Welcome-card-1",
      "media": "Welcome-media-2",
    }
  }
/>
`;

exports[`Welcome renders component 2`] = `
Object {
  "classes": Object {
    "card": "Welcome-card-1",
    "media": "Welcome-media-2",
  },
}
`;
